File: demo1/src/main/java/org/mosquito/demo/service/IUserService.java
package org.mosquito.demo.service;

import org.mosquito.framework.jdbc.service.IBaseService;
import org.mosquito.demo.model.User;
import org.mosquito.demo.dto.UserDto;

/**
 * 业务类接口
 *
 * @author jcchen
 * @Date 2018-5-26
 * 
 */
public interface IUserService extends IBaseService<UserDto, User>{
	
}

File: demo1s/src/main/scala/org/mosquito/demo1s/service/IUserService.scala
package org.mosquito.demo1s.service

import org.mosquito.framework.jdbc.service.{IBaseService, ScalaSupport}
import org.mosquito.demo1s.model.User
import org.mosquito.demo1s.dto.UserDto

/**
 * 业务类接口
 *
 * @author jcchen
 * @Date 2018-5-28
 * 
 */
trait IUserService extends IBaseService[UserDto, User] with ScalaSupport[UserDto, User] {
	
}
